client/watch.html

```diff
@@ -731,7 +735,7 @@
 </template>
 
 <template name="list_item_context_section">
-  <div class="list-item-context-plus-annotation" data-context-id="{{_id}}">
+  <div class="list-item-context-plus-annotation" data-context-id="{{_id}}" style="{{#unless showContext}} display:none;{{/unless}}">
```

Comment:
Yes I've had bad experiences with maps in this case before. The exact presentation or how to bring the items into the list on cue hasn't been defined yet -- in the meantime I'll switch back to your suggestion